src/static/riot/tasks/management.tag

```diff
@@ -98,6 +95,68 @@
         </table>
     </div>
 
+    <div class="ui modal" ref="modal">
+        <div class="header">
+            Create Task
+        </div>
+        <div class="content">
+            <div class="ui pointing menu">
+                <div class="active item modal-item" data-tab="details">Details</div>
+                <div class="item modal-item" data-tab="data">Datasets</div>
+            </div>
+            <form class="ui form" ref="form">
+                <div class="ui active tab" data-tab="details">
+                    <div class="required field">
+                        <label>Name</label>
+                        <input name="name" placeholder="Name" ref="name" onkeyup="{ form_updated }">
+                    </div>
+                    <div class="required field">
+                        <label>Description</label>
+                        <textarea rows="4" name="description" placeholder="Description" ref="description" onkeyup="{ form_updated }"></textarea>
+                    </div>
+                </div>
+                <div class="ui tab" data-tab="data">
+                    <div>
+                        <div class="two fields" data-no-js>
+                            <div class="field {required: file_field === 'scoring_program'}"
+                                 each="{file_field in ['scoring_program', 'ingestion_program']}">
+                                <label>
+                                    {_.startCase(file_field)}
+                                    <span data-tooltip="Something useful to know...!" data-inverted=""
```

Comment:
fix indentation here?

Reply:
Fix indentation how? it looks right to me?

Reply:
```html
<span data-tooltip="Something useful to know...!" data-inverted="" data-position="bottom center">
    <i class=" grey help icon circle"></i>
</span>
```

Reply:
OR

```html
<span data-tooltip="Something useful to know...!" 
      data-inverted="" 
      data-position="bottom center">
    <i class=" grey help icon circle"></i>
</span>
```